### PR TITLE
 HACKING.md: Elaborate on testing, interacting with PRs

### PR DESCRIPTION
**Pull request by cgwalters**
Useful knowledge I want to write down for new team members.

### CODE REVIEW

**Comment by runcom** [comment]
omg amazing

/approve

File: docs/HACKING.md [hunk 1]
```diff
@@ -157,3 +157,38 @@ the operator since the MCD daemonset will be overwritten:
...
+
+In the `pods/` directory in "artifacts" you can find the logs for the various
+components.  Also you can see the node annotations in `nodes.json`; usually
+the `machineconfigs.json` and `machineconfigpools.json` are also there too.
+
+## Logging into a cluster live
+
+Login: `oc login https://api.ci.openshift.org/console/`
+
+In the pull request, you'll see a "project" or Kubernetes namespace, so you can e.g.: `oc project ci-op-zgctgrpy`
```
Comment (kikisdeliveryservice) [inline docs/HACKING.md]: Where is this in the PR? I've never seen it? Can we add a picture or something?

Comment (cgwalters) [reply docs/HACKING.md]: Updated :arrow_down:

Comment (kikisdeliveryservice) [reply docs/HACKING.md]: so colin helped me find this, but it's quite hidden so i think we can be explicity about where it is located, ie click on the aws e2e test - click on raw build log - show some example lines where it is.

Comment (kikisdeliveryservice) [reply docs/HACKING.md]: perf!

**Pull request merged**